src/utils/data/animals.mock.json

```diff
@@ -1,89 +1,67 @@
 {
   "edges": [
+    { "source": "9d989e8d", "target": "f42dab68" },
+    { "source": "85c4cf64", "target": "f42dab68" },
+    { "source": "0ae9e4de", "target": "f42dab68" },
+    { "source": "09f5edeb", "target": "f42dab68" },
+    { "source": "f42dab68", "target": "2cd4ba93" },
+    { "source": "f42dab68", "target": "fac8f1a3" },
+    { "source": "e4951252", "target": "443cf06a" },
+    { "source": "c38d4c6a", "target": "443cf06a" },
+    { "source": "f1f1425b", "target": "443cf06a" },
+    { "source": "9d989e8d", "target": "443cf06a" },
+    { "source": "443cf06a", "target": "6525f2e6" },
+    { "source": "443cf06a", "target": "091b5035" },
+    { "source": "6525f2e6", "target": "689862f0" },
+    { "source": "689862f0", "target": "4bbf2ddc" },
+    { "source": "2cd4ba93", "target": "e27376a9" },
+    { "source": "6525f2e6", "target": "e27376a9" },
+    { "source": "e27376a9", "target": "1769e230" }
+  ],
+  "layers": ["raw", "feature", "primary", "model output"],
+  "modular_pipelines": [
     {
-      "source": "e4951252",
-      "target": "443cf06a"
-    },
-    {
-      "source": "c38d4c6a",
-      "target": "443cf06a"
-    },
-    {
-      "source": "f1f1425b",
-      "target": "443cf06a"
-    },
-    {
-      "source": "9d989e8d",
-      "target": "443cf06a"
-    },
-    {
-      "source": "443cf06a",
-      "target": "6525f2e6"
-    },
-    {
-      "source": "443cf06a",
-      "target": "091b5035"
-    },
-    {
-      "source": "9d989e8d",
-      "target": "f42dab68"
-    },
-    {
-      "source": "85c4cf64",
-      "target": "f42dab68"
-    },
-    {
-      "source": "0ae9e4de",
-      "target": "f42dab68"
-    },
-    {
-      "source": "09f5edeb",
-      "target": "f42dab68"
-    },
-    {
-      "source": "f42dab68",
-      "target": "2cd4ba93"
-    },
-    {
-      "source": "f42dab68",
-      "target": "fac8f1a3"
+      "id": "pipeline1",
+      "name": "Pipeline1"
     },
     {
-      "source": "2cd4ba93",
-      "target": "e27376a9"
+      "id": "pipeline1.data_engineering",
```

Comment:
Hm, I think we might've not foreseen this in the ticket description... Having `pipeline1` as well here is probably quite useful for the UX in the frontend  (despite not directly assigned to any node). I suppose we can reiterate on that once we have this one merged and just keep a note when we can see things in the frontend.

Reply:
It's just above (looks a bit odd through the way git is showing the diff)